### ISSUE TITLE
No handling of gzipped genome fasta file

### ISSUE THREAD

**Issue by balwierz** [vaquerizaslab/fanc#1]
This step happens after sam sorting

```Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/fanc/regions.py", line 106, in genome_regions
    regions = gr_load(re_or_file)
  File "/usr/local/lib/python3.7/dist-packages/genomic_regions/regions.py", line 190, in load
    raise ValueError("File type not recognised ({}).".format(file_name))
ValueError: File type not recognised (genome.fa.gz)
```

**Comment by kaukrise** [comment]
Hi, thanks for pointing this out. I'll see if I can add gzipped FASTA support for the next version. For now, simply unzip the file and you should be good to go!

**Comment by kaukrise** [comment]
Fixed in 0.8.18
